### PR TITLE
Fix webpack-related error when using in browser

### PR DESCRIPTION
**Pull request by kennym**
I had to re-build the output files in dist/ using `webpack` and then `gulp build:optimize` - after that everything worked.

### CODE REVIEW

**Comment by kennym** [comment]
The failing Travis CI build doesn't seem to be my fault after reading the log: https://travis-ci.org/GetStream/stream-js/jobs/121268370

Seems to be some intermittent error.

**Comment by matthisk** [comment]
So what was the error you were experiencing with the old distributable file and how did you include the package on your frontend?

**Comment by matthisk** [comment]
BTW I think we need to do a new patch release for the distributable to be fixed on Bower as well as inside the repository.

**Comment by matthisk** [comment]
Created a new release 3.2.1, everything seems to be working fine again.

File: dist/js/getstream.js [hunk 1]
```diff
@@ -62,8 +62,8 @@ return /******/ (function(modules) { // webpackBootstrap
 	'use strict';
 
 	var StreamClient = __webpack_require__(2);
-	var errors = __webpack_require__(4);
-	var request = __webpack_require__(!(function webpackMissingModule() { var e = new Error("Cannot find module \"request\""); e.code = 'MODULE_NOT_FOUND'; throw e; }()));
```
Comment (kennym) [inline dist/js/getstream.js]: @matthisk I would get this error in the console

**Comment by kennym** [comment]
Thanks. It's fixed now. 👍🏼

**Pull request closed without merging**